acceptance/tests/tasks/run_ps1.rb

```diff
@@ -20,15 +20,13 @@
 
   step 'Create powershell task on agent hosts' do
     windows_hosts.each do |agent|
-      create_task_on(agent, 'echo', 'init.ps1', <<-EOF)
-foreach ($i in $input) { Write-Output $i }
-Write-Output $env:PT_data
-EOF
+      task_body = "foreach ($i in $input) { Write-Output $i }\nWrite-Output $env:PT_data";
```

Comment:
All of these could go back to using HEREDOC notation (assigning to @sha256 would be the only change).